spec/spec_helper.rb

```diff
@@ -98,6 +98,8 @@ def Object.method_added(method)
     # a real object. This is generally recommended, and will default to
     # `true` in RSpec 4.
     mocks.verify_partial_doubles = true
+    # Prevents expect(nil).to receive(...)
+    mocks.allow_message_expectations_on_nil = false
```

Comment:
This should be in a separate commit with it's own justification. Which expectation needs this? Is there an alternative approach that avoids the need to add it?